### PR TITLE
Move hex address computation under lisk32 computation

### LINKED ISSUE #1816: Remove content regarding `hex` address
### Description
Following updates in the latest SDKv6, we need to remove all mentions of hex address in the docs, as it is not used anymore inside the Lisk SDK.

#### Affected branch / URL
`development`

### Additional resources
https://github.com/LiskHQ/lisk-sdk/issues/7469

### PR DESCRIPTION
**Pull request by Tschakki**
Closes #1816

### CODE REVIEW

File: docs/modules/ROOT/pages/run-blockchain/account-management.adoc [hunk 1]
```diff
@@ -422,27 +426,17 @@ First, create the SHA-236 hash of the public key:
...
+===== Obtain Lisk32 address from hex address
+The Lisk32 representation can be obtained from a hex address as follows:
 
-. The <<hexadecimal-representation,hex address>> is used to create a 30-bit checksum of the address, using a BCH code{fn_bip173}.
+. The hex address is used to create a 30-bit checksum of the address, using a BCH code{fn_bip173}.
 This step provides protection against accidental typing mistakes.
 . Concatenate the address and the output of step 1.
-. Encode the output of step 2 in a custom Base32 format, see {url_github_lip_18}[LIP 0018^] for details.
+. Encode the output of step 2 in a custom Base32 format, see {url_github_lip_18_base32}[LIP 0018^] for details.
 Lower-case letters and digits are used, and to improve usability the characters i, l, 1 and 0 have been excluded.
```
Comment (chris529) [inline docs/modules/ROOT/pages/run-blockchain/account-management.adoc]:
```suggestion
Lowercase letters and digits are used, and to improve usability the characters i, l, 1, and 0 have been excluded.
```

File: docs/modules/ROOT/pages/run-blockchain/account-management.adoc [hunk 2]
```diff
@@ -422,27 +426,17 @@ First, create the SHA-236 hash of the public key:
...
-Each address starts with the prefix “lsk" and a checksum is appended.
-
-.Lisk32 address example
- lsk24cd35u4jdq8szo3pnsqe5dsxwrnazyqqqg5eu
-
-The *Lisk32 representation* displays addresses as a checksum with the "lsk" prefix to prevent accidental typing mistakes.
-
-==== Lisk32 address computation
-The procedure to obtain the Lisk32 representation of a <<hexadecimal-representation>> of an address is as follows:
+===== Obtain Lisk32 address from hex address
```
Comment (chris529) [inline docs/modules/ROOT/pages/run-blockchain/account-management.adoc]:
```suggestion
===== Obtain the Lisk32 address from the hex address
```

File: docs/modules/ROOT/pages/run-blockchain/account-management.adoc (hunk 1 above)
Comment (chris529) [reply docs/modules/ROOT/pages/run-blockchain/account-management.adoc]: In American English it should really be "Lowercase" without the hyphen.

**Review by chris529 — APPROVED** [review]
LGTM! 👍  Looks great, I have just left some minor suggestions.

File: docs/modules/ROOT/pages/run-blockchain/account-management.adoc [hunk 3]
```diff
@@ -402,18 +403,21 @@ The user substore contains the following entries:
 ----
 ====
 
-== Account address formats
+== Lisk32 address format
+The *Lisk32 representation* displays addresses as a checksum with the "lsk" prefix to prevent accidental typing mistakes.
```
Comment (sergeyshemyakov) [inline docs/modules/ROOT/pages/run-blockchain/account-management.adoc]: A checksum is only the last 6 letters of the address, so it is a bit incorrect to say that Lisk32 displays addresses as a checksum. I would say: "... displays addresses as a readable string with a checksum and the "lsk" prefix..."

File: docs/modules/ROOT/pages/run-blockchain/account-management.adoc [hunk 4]
```diff
@@ -402,18 +403,21 @@ The user substore contains the following entries:
...
 
-The account address can be displayed in different formats, depending on the context.
+.Automatic detection of address typing errors
+NOTE: A BCH checksum is used to check for {url_github_lip_18}[detection of small errors^] in the address.
```
Comment (sergeyshemyakov) [inline docs/modules/ROOT/pages/run-blockchain/account-management.adoc]: I would remove "to check" from this sentence.

File: docs/modules/ROOT/pages/run-blockchain/account-management.adoc [hunk 5]
```diff
@@ -422,28 +426,18 @@ First, create the SHA-236 hash of the public key:
```
Comment (sergeyshemyakov) [inline docs/modules/ROOT/pages/run-blockchain/account-management.adoc]: In the line above, replace "SHA-236" with "SHA-256".

File: docs/modules/ROOT/pages/run-blockchain/account-management.adoc [hunk 6]
```diff
@@ -422,28 +426,18 @@ First, create the SHA-236 hash of the public key:
...
-Each address starts with the prefix “lsk" and a checksum is appended.
-
-.Lisk32 address example
- lsk24cd35u4jdq8szo3pnsqe5dsxwrnazyqqqg5eu
-
-The *Lisk32 representation* displays addresses as a checksum with the "lsk" prefix to prevent accidental typing mistakes.
-
-==== Lisk32 address computation
-The procedure to obtain the Lisk32 representation of a <<hexadecimal-representation>> of an address is as follows:
+===== Obtain the Lisk32 address from the hex address
```
Comment (sergeyshemyakov) [inline docs/modules/ROOT/pages/run-blockchain/account-management.adoc]: I would suggest you continue the example from the LIP (at least put the final Lisk32 representation of the address `lsk24cd35u4jdq8szo3pnsqe5dsxwrnazyqqqg5eu`).

File: docs/modules/ROOT/pages/run-blockchain/account-management.adoc [hunk 7]
```diff
@@ -13,7 +13,8 @@ Mona Bärenfänger <mona@lightcurve.io>
 :url_lisk_wallet: https://lisk.com/wallet
 :url_github_bip_39: https://github.com/bitcoin/bips/blob/master/bip-0039.mediawiki#generating-the-mnemonic
 :url_github_bip_173: https://github.com/bitcoin/bips/blob/master/bip-0173.mediawiki
-:url_github_lip_18: https://github.com/LiskHQ/lips/blob/master/proposals/lip-0018.md#step-3-base32
+:url_github_lip_18: https://github.com/LiskHQ/lips/blob/master/proposals/lip-0018.md
```
Comment (sergeyshemyakov) [inline docs/modules/ROOT/pages/run-blockchain/account-management.adoc]: These links go to the `master` branch, which does not exist in LIPs repo. Change to `main`.

**Review by sergeyshemyakov — CHANGES_REQUESTED** [review]
Some comments from my side.

File: docs/modules/ROOT/pages/run-blockchain/account-management.adoc (hunk 6 above)
Comment (Tschakki) [reply docs/modules/ROOT/pages/run-blockchain/account-management.adoc]: I removed now explanations now, and added a direct link to the explanations in the LIP

**Review by sergeyshemyakov — APPROVED** [review]

**Pull request merged**